src/plugins/translation-xml.js

```diff
@@ -0,0 +1,32 @@
+const _ = require('lodash');
+const { createFilter } = require('rollup-pluginutils');
+const parser = require('xml2json');
+
+module.exports = (opts = {}) => {
```

Comment:
You should do: 

```javascript
({ exclude, include = '**/Messages_*.xml' })
```

Reply:
except that opts might be null and you can't destructure null or undefined values